### PR TITLE
Extend the SYSMAN Frequency Domain list to include a MEDIA Domain

### PR DESCRIPTION
**Pull request by pmmeyer**
Addresses #60.

### CODE REVIEW

**Comment by jandres742** [comment]
@pmmeyer : thanks. This addresses https://github.com/oneapi-src/level-zero-spec/issues/60, correct?

@sanilkumar0 , @saik-intel: could you take a look?

**Comment by pmmeyer** [comment]
> @pmmeyer : thanks. This addresses #60, correct?
> 
> @sanilkumar0 , @saik-intel: could you take a look?

Yes, that is correct.

**Comment by ezhilintc** [comment]
> > @pmmeyer : thanks. This addresses #60, correct?
> > @sanilkumar0 , @saik-intel: could you take a look?
> 
> Yes, that is correct.

This addresses spec side of it. We still need some changes in sysman UMD, right?

**Comment by jandres742** [comment]
@ezhilintc : Correct. Once these changes are merged and released, we could start implementation in driver.

I think this could be added in next minor version, v1.6.

**Review by ezhilintc — APPROVED** [review]

**Comment by pmmeyer** [comment]
@jandres742 - will someone be merging this PR? I don't seem to have permission.

**Comment by jandres742** [comment]
@pmmeyer thanks. We haven't merged this PR since we are targeting this for v1.6, since it is a change in enum. Before that, we are thinking on having another update, 1.5.X, so we will merge this afterwards. After merging, it will be released on 1.6 loader, and then absorbed by driver implementations.

Let me know if this helps.

**Review by jandres742 — CHANGES_REQUESTED** [review]
To be merged for v1.6.

File: scripts/sysman/frequency.yml [hunk 1]
```diff
@@ -19,6 +19,8 @@ etors:
       desc: "GPU Core Domain."
     - name: MEMORY
       desc: "Local Memory Domain."
+    - name: MEDIA
```
Comment (bmyates) [inline scripts/sysman/frequency.yml:22]: Add a `version: 1.6` guard to new addition

**Comment by bmyates** [comment]
Code targeting 1.6 should be mergeable if it is properly guarded with version checks

**Comment by jandres742** [comment]
> Code targeting 1.6 should be mergeable if it is properly guarded with version checks

thanks @bmyates . You are correct. @pmmeyer : could you add the versioning?

**Comment by wdamon-intel** [comment]
> could you add the versioning

@pmmeyer please follow up with the versioning tags.

**Review by jandres742 — APPROVED** [review]

**Pull request merged**